index.html

```diff
@@ -4,72 +4,133 @@
+          <h5>Linux</h5>
           <a href="https://snapcraft.io/microk8s" title="Get it from the Snap Store">
             <img src="https://snapcraft.io/static/images/badges/en/snap-store-white.svg" alt="Get it from the Snap Store" width="200" />
           </a>
         </div>
-        <div class="col-4 prefix-1 u-vertically-center">
-          <img src="https://assets.ubuntu.com/v1/96d35aa4-certified-kubernetes-logo.svg" alt="Certified Kubernetes" />
+        <div class="col-4 p-divider__block u-align--center">
+          <h5>Windows</h5>
+          <p>
+            <a href="https://multipass.run/">
+            <img src="https://assets.ubuntu.com/v1/ea34f006-Multipass+logomark_rgb.svg" width="62" />
+            <br />
+            Run it with Multipass
+            </a>
+          </p>
+        </div>
+        <div class="col-4 p-divider__block u-align--center">
+          <h5>macOS</h5>
+          <p>
+            <a href="https://multipass.run/">
+            <img src="https://assets.ubuntu.com/v1/ea34f006-Multipass+logomark_rgb.svg" width="62" />
+            <br />
+            Run it with Multipass
+            </a>
+          </p>
         </div>
       </div>
     </section>
 
-    <section class="p-strip is-shallow is-bordered">
-      <div class="row">
-        <h2 class="p-muted-heading">Features</h2>
+    <section class="p-strip--light">
+      <div class="row u-vertically-center">
+        <div class="col-6">
+          <blockquote  class="p-pull-quote">
+            <p>
+              “Canonical might have assembled the easiest way to provision a single node Kubernetes cluster”
+            </p>
+            <cite class="p-pull-quote__citation">
+              &mdash; Kelsey Hightower
+            </cite>
+          </blockquote>
+        </div>
+        <div class="col-6">
+          <blockquote class="p-pull-quote">
+            <p>
+              “dear microk8s team: you are awesome. Microk8s has tangibly improved my life and my CI pipelines as a developer of k8s controllers”
+            </p>
+            <cite class="p-pull-quote__citation">
+              &mdash; in <a href="https://kubernetes.slack.com/messages/microk8s/" class=" p-link--external">#microk8s</a>
+            </cite>
+          </blockquote>
+        </div>
       </div>
+    </section>
+    <section class="p-strip is-bordered">
       <div class="row u-equal-height">
-        <div class="col-3">
+        <div class="col-2 col-start-large-2">
           <ul class="p-list--divided">
-            <li class="p-list__item is-ticked">1-click Istio, Knative, Fluentd, Linkerd</li>
-            <li class="p-list__item is-ticked">1-click Jaeger, Prometheus, and more </li>
-            <li class="p-list__item is-ticked">Clustering</li>
+            <li class="p-list__item is-ticked">Conformant</li>
+            <li class="p-list__item is-ticked">Istio</li>
+            <li class="p-list__item is-ticked">Clustering<br /><small style="color: red;">BETA</small></li>
```

Comment:
Maybe be the first to use the brand new labels components: `<small class="p-label--in-progress">BETA</small>` There are different version. https://docs.vanillaframework.io/patterns/labels/